### PR TITLE
Add descriptions to remaining properties of RDSamplerState

### PR DESCRIPTION
**Pull request by DevPoodle**
I added descriptions to the min_filter and unnormalized_uvw properties of RDSamplerState. I also updated the mag_filter description to be a little more helpful and consistent with the new min_filter description.

### CODE REVIEW

**Review by Mickeon — APPROVED** [review]
I like the wording here. I cannot speak for factual accuracy but it does look correct.

**Comment by reduz** [comment]
Factual accuracy is good

File: doc/classes/RDSamplerState.xml [hunk 1]
```diff
@@ -26,12 +26,13 @@
 			The mipmap LOD bias to use. Positive values will make the sampler blurrier at a given distance, while negative values will make the sampler sharper at a given distance (at the risk of looking grainy). Recommended values are between [code]-0.5[/code] and [code]0.0[/code]. Only effective if the sampler has mipmaps available.
 		</member>
 		<member name="mag_filter" type="int" setter="set_mag_filter" getter="get_mag_filter" enum="RenderingDevice.SamplerFilter" default="0">
-			The sampler's magnification filter.
+			The sampler's magnification filter. It is the filtering method used when sampling a texture at a higher resolution.
```
Comment (reduz) [inline doc/classes/RDSamplerState.xml:29]: you could add when a texel apppears bigger than a pixel.

File: doc/classes/RDSamplerState.xml [hunk 2]
```diff
@@ -26,12 +26,13 @@
...
 		</member>
 		<member name="max_lod" type="float" setter="set_max_lod" getter="get_max_lod" default="1e+20">
 			The maximum mipmap LOD bias to display (lowest resolution). Only effective if the sampler has mipmaps available.
 		</member>
 		<member name="min_filter" type="int" setter="set_min_filter" getter="get_min_filter" enum="RenderingDevice.SamplerFilter" default="0">
+			The sampler's minification filter. It is the filtering method used when sampling a texture at a lower resolution.
```
Comment (reduz) [inline doc/classes/RDSamplerState.xml:35]: when a texel is smaller than a pixel

**Review by reduz — COMMENTED** [review]
some bits to make it clearer, though i guess whoever is using this API knows this..

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**